code/modules/projectiles/ammo_datums.dm

```diff
@@ -1358,8 +1358,8 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	damage = 20
 	penetration = 10
 	sundering = 1
-	max_range = 40
-	damage_falloff = 0
+	max_range = 30
+	damage_falloff = 0.2 ///Doesn't seem to even apply when fired out of the laser gun.
```

Comment:
then don't change it? You probably have to change it for each fire mode datum of the lasrifle

Reply:
No, just the base is enough which I already did in the PR.